packages/gcf-utils/src/bin/README.md

```diff
@@ -47,6 +47,7 @@ Application you wish to use with your bot.
  export BUCKET=<NAME OF BUCKET TO STORE KEYS>
  export BOT=<NAME OF YOUR BOT>
  ```
+4. Install dependencies and link using npm i and npm link
 
 5. Run the `genkey` tool
```

Comment:
nit: Numbers need to increment
4 -> 5
5 -> 6

Reply:
ah, yes, great catch! Thanks!